src/st/prim_helpers.h

```diff
@@ -183,8 +185,13 @@ Primitive* PrimToggleVisibility(Primitive* firstPrim, s32 count) {
         }
 
         prim = prim->next;
-        if (prim == NULL)
-            return 0;
+        if (prim == NULL) {
+            // Required for PSP match
+            if (true) {
```

Comment:
Another weird one needed for registers, there's a few things that work here, such as wrapping this as `do { return NULL } while(0);` or even `return NULL; do {} while(0);` after the return matches PSP. 

This seems the "least weird" to me, but I can change it if there's a fix or preference
https://decomp.me/scratch/8l5Dr